server/application-server/src/main/java/de/tum/in/www1/hephaestus/codereview/pullrequest/PullRequestConverter.java

```diff
@@ -17,13 +17,17 @@ public class PullRequestConverter extends BaseGitServiceEntityConverter<GHPullRe
 
     @Override
     public PullRequest convert(@NonNull GHPullRequest source) {
-        IssueState state = convertState(source.getState());
         PullRequest pullRequest = new PullRequest();
-        convertBaseFields(source, pullRequest);
         pullRequest.setNumber(source.getNumber());
-        pullRequest.setTitle(source.getTitle());
         pullRequest.setUrl(source.getHtmlUrl().toString());
```

Comment:
Why not move the two lines above to update as well?

Reply:
My assumption was that the PR number is unique and it never changes throughout the lifecycle of a PR. So there should be no need to ever update it. The URL is just the combination of the repository `nameWithOwner` and the PR number, so same logic would hold here.

Reply:
Makes sense 👍 technically the `nameWithOwner` can maybe change when you transfer the repo but that is an edge case 🤔